_integration-schemas/facebook-ads/ads_insights_hourly_advertiser.md

```diff
@@ -0,0 +1,348 @@
+---
+tap: "facebook"
+version: "1"
+key: ""
+
+name: "ads_insights_hourly_advertiser"
+doc-link: "https://developers.facebook.com/docs/marketing-api/insights/fields/"
+singer-schema: "https://github.com/singer-io/tap-facebook/blob/master/tap_facebook/schemas/ads_insights_hourly_advertiser.json"
+description: |
+  The `ads_insights_hourly_advertiser` table contains entries for each campaign/set/ad combination for each day, along with detailed statistics, segmented by audience and advertiser time zones.
+  This table contains the same fields as the [`ads_insights`](#ads_insights) table, with the exception of `country`.
```

Comment:
Looks like you've got some copypasta here - notably, `country`. This should be `hourly_stats_aggregated_by_advertiser_time_zone`.